doc/sphinx/Pacemaker_Administration/troubleshooting.rst

```diff
@@ -58,16 +88,36 @@ state at that moment (the CIB). This file can help determine why particular
 actions were scheduled. The ``crm_simulate`` command, described in
 :ref:`crm_simulate`, can be used to replay the file.
 
-Further Information About Troubleshooting
-#########################################
+The log messages immediately before the "saving inputs" message will include
+any actions that the scheduler thinks need to be done.
+
+
+Node Failures
+#############
+
+When a node fails, and looking at errors and warnings doesn't give an obvious
+explanation, try to answer questions like the following based on log messages:
+
+* When and what was the last successful message on the node itself, or about
+  that node in the other nodes' logs?
+* Did pacemaker-controld on the other nodes notice the node leave?
+* Did pacemaker-controld on the DC invoke the scheduler and schedule a new
+  transition?
+* Did the transition include fencing the failed node?
+* Was the fencing attempted?
+* Did the fencing succeed?
```

Comment:
I think these two read more naturally as "Was fencing attempted?" and "Did fencing succeed?"

Reply:
thanks